script.js

```diff
@@ -0,0 +1,23 @@
+const expansesContainer = document.querySelector('.expenses-container');
+const totalDisplay = document.getElementById('total');
+const API = 'http://localhost:3000/api/list/';
+
+async function fetchAPI() {
+    const url = await fetch("http://localhost:3000/api/list");
```

Comment:
why you still describe your api url like static string if you got it in separate global variable ?